build.sbt

```diff
@@ -10,6 +10,12 @@ lazy val backend = (project in file("backend"))
 lazy val engine = (project in file("engine"))
   .dependsOn(core % "test->test;compile->compile", backend)
   .settings(engineSettings:_*)
+  .configs(AllTests).settings(inConfig(AllTests)(Defaults.testTasks): _*)
```

Comment:
I made a comment on the PR but ideally we wouldn't need this boilerplate on every subproject which uses it. However I need to fix this problem sooner than I have time to investigate the abstraction